CASMINST-7035 remove docs for manual CSM upgrade in 1.6 (#5510)
Cray-HPE/docs-csm: operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md

1. CSM preparation

   Read the _Important Notes_ section of the
   [CSM 1.5.0 or later to 1.6.0 Upgrade Process](../../../upgrade/Upgrade_Management_Nodes_and_CSM_Services.md)
   documentation and then follow only these CSM instructions in order:

   1. [Prepare for Upgrade](../../../upgrade/prepare_for_upgrade.md)

1. Prepare for the upgrade procedure and download product media

   1. Follow the IUF [Prepare for the install or upgrade](preparation.md) instructions to set
      environment variables used during the upgrade process.

   1. Download the desired HPE product media defined by the HPC CSM Software Recipe to `${MEDIA_DIR}`, which was defined in the previous step.

1. Product delivery

   > **NOTE** The CSM upgrade prerequisites are automatically executed in a hook run before `pre-install-check`.

   Follow the IUF [Product delivery](product_delivery.md) instructions.

1. Configuration

   Follow the IUF [Configuration](configuration.md) instructions.

1. Image preparation

   Follow the IUF [Image preparation](image_preparation.md) instructions.

1. Backup

   Follow the IUF [Backup](backup.md) instructions.

1. Management rollout

>    > **NOTE** The CSM services upgrade and validate CSM health are automatically executed in a hook run before the first management node is rolled out.

Reviewer: Awkward wording. How about:

```suggestion
   > **NOTE** The upgrade of CSM services and validation of CSM health occur automatically in a hook executed before the first management node is rolled out.
```

1. Deploy product

   > **NOTE**  Networking and CoreDNS anti-affinity are applied and the Kubernetes control plane is upgraded in a hook automatically executed at the end of `deploy-product`.

   Follow these IUF instructions in order:

   1. [Deploy product](deploy_product.md)
   1. [Validate deployment](validate_deployment.md)

1. Managed rollout

   Follow the IUF [Managed rollout](managed_rollout.md) instructions.

The IUF upgrade workflow is now complete. Exit any typescript sessions created during the upgrade
procedure and remove any installation artifacts, if desired.

## Description of CSM upgrade hooks

The hooks below are automatically executed when CSM is being upgraded with IUF.

- CSM upgrade prerequisites

   CSM upgrade prerequisites are executed in a hook run before `pre-install-check`. This executes steps that are dependencies for later CSM upgrade steps.
   This includes some service chart upgrades, uploading base NCN images to be used later in `prepare-images`, and other setup steps.
   The specific script that is being executed is `/usr/share/doc/csm/upgrade/scripts/upgrade/prerequisites.sh`.

> - CSM services upgrade and validate CSM health

Reviewer: Another awkward wording instance.

```suggestion
- Upgrade of CSM services and validation of CSM health
```

>    CSM services upgrade and validate CSM health are executed in a hook run before `management-nodes-rollout`. This hook is only executed before the first NCN is upgraded.

Reviewer: ```suggestion
   The upgrade of CSM services and validation of CSM health are performed in a hook executed before `management-nodes-rollout`. This hook is only executed before the first NCN is upgraded.
```

   The specific script that executes the CSM services upgrade is `/usr/share/doc/csm/upgrade/scripts/upgrade/csm-upgrade.sh`.

> - Apply networking, apply CoreDNS anti-affinity, and upgrade Kubernetes control plane

Reviewer: What does "Apply networking" mean? Maybe "apply networking changes"? I'd also make it a noun so each item of this list is a consistent part of speech.

```suggestion
- Application of networking changes, CoreDNS anti-affinity, upgrade of the Kubernetes control plane
```

>    Apply networking, apply CoreDNS anti-affinity, and completing the Kubernetes control plane upgrade is executed in a hook run after `deploy-product`.

Reviewer: ```suggestion
   The application of networking changes and CoreDNS anti-affinity changes along with the upgrade of the Kubernetes control plane is performed in a hook executed after `deploy-product`.
```
